.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - *restore-cache
       - run:
           name: Run cargo clippy
-          command: cargo clippy --all-targets --all-features --workspace -- -D warnings
+          command: cargo clippy --all-features --workspace -- -D warnings
```

Comment:
Was that change intentional? I think checking tests and examples is a good idea.

Reply:
If running a nightly Clippy isn't a problem, the fix is `cargo +<< pipeline.parameters.nightly-version >> clippy --all-targets --all-features --workspace -- -D warnings` (and removing the `--profile minimal from the nightly installation`.

Reply:
that is probably a better solution